restli-tools/src/main/resources/apiVmTemplates/action.vm

```diff
@@ -23,24 +23,14 @@ Copyright (c) 2021 LinkedIn Corp.
     @SuppressWarnings("unchecked")
     public CompletionStage<${method.valueClassDisplayName}> ${util.nameCamelCase(${method.name})}(
       #if(${method.isEntityAction()})
-      $spec.keyClassName $spec.idName,
+        $spec.keyClassName $spec.idName,
```

Comment:
this line should also be guarded by "$method.hasActionParams()"

Reply:
The hasRequiredParams check in line 22 ensures there will be params.